### PR TITLE
support older and future versions of bundler that do not have clean_env

### PR DESCRIPTION
**Pull request by grosser**
fixes https://github.com/puma/puma/issues/1312

### CODE REVIEW

**Comment by grosser** [comment]
@nateberkopec @matobinder

File: lib/puma/launcher.rb [hunk 1]
```diff
@@ -163,7 +163,7 @@ def phased_restart
 
     # Run the server. This blocks until the server is stopped
     def run
-      previous_env = (defined?(Bundler) ? Bundler.clean_env : ENV.to_h)
+      previous_env = (defined?(Bundler) ? Bundler::ORIGINAL_ENV : ENV.to_h)
```
Comment (nateberkopec) [inline lib/puma/launcher.rb]: Why not `.original_env`?

Comment (nateberkopec) [inline lib/puma/launcher.rb]: Ah I see, nvm. Should we be `clone`ing this like Bundler does though?

**Comment by grosser** [comment]
we don't modify it, so no

**Comment by grosser** [comment]
just being afraid that's not defined again on some version

**Pull request merged**